packaging/grafana-agent-flow/grafana-agent-flow.river

```diff
@@ -0,0 +1,27 @@
+// Sample config for Grafana Agent Flow.
+//
+// For a full configuration reference, see https://grafana.com/docs/agent/latest/flow/
+logging {
+	level = "warn"
+}
+
+prometheus.exporter.unix {
+	include_exporter_metrics = true
+	disable_collectors       = ["mdadm"]
+}
+
+prometheus.scrape "default" {
+	targets = concat(
+		prometheus.exporter.unix.targets,
+		[{
+			// Self-collect metrics
+			job         = "agent",
+			__address__ = "127.0.0.1:12345",
+		}],
+	)
+
+	forward_to = [
+	// TODO: components to forward metrics to (like prometheus.remote_write or
+	// prometheus.relabel).
```

Comment:
This is what the formatter is doing, even though I don't think this is where the comments should go :disappointed: